workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -11,15 +11,17 @@ public class EntityGameObjectLinker
         private readonly World world;
         private readonly HashSet<Type> gameObjectComponentTypes = new HashSet<Type>();
         private readonly ILogDispatcher logDispatcher;
+        private readonly GameObjectDispatcherSystem gameObjectDispatcherSystem;
 
         public EntityGameObjectLinker(World world, ILogDispatcher logDispatcher)
         {
             this.world = world;
             this.logDispatcher = logDispatcher;
+            gameObjectDispatcherSystem = world.GetOrCreateManager<GameObjectDispatcherSystem>();
         }
 
         public void LinkGameObjectToEntity(GameObject gameObject, Entity entity, EntityId spatialEntityId,
-            ViewCommandBuffer viewCommandBuffer)
+            EntityCommandBuffer entityCommandBuffer, ViewCommandBuffer viewCommandBuffer)
```

Comment:
why do we need to add two buffers?

Reply:
One is for adding Componentdatas and one is for adding non-blittable components.

Reply:
This looks very confusing for an API users should be able to access and use too. I don't have much context on these buffers - could we have the Linker own them maybe and have a flush method?